docs/moz-addon-manager.md

```diff
@@ -0,0 +1,173 @@
+# Develop features for mozAddonManager
+
+The AMO and Discovery Pane apps use the
+[`mozAddonManager`](https://bugzilla.mozilla.org/show_bug.cgi?id=1310752)
+web API to achieve a more seamless add-on installation user experience.
+However, this API is only available to a limited list of domains.
+If you go to https://addons.mozilla.org (production) in Firefox then `mozAddonManager`
+is available on the page. If you go to a development site then it's not.
+
+## Turning on mozAddonManager for development
```

Comment:
These docs don't appear to mention installation. I'm guessing that will implicitly work if you're installing from prod (e.g. you're using prod apis) but if the apis are pointing anywhere else will require additional steps.